docs/concepts/realms.md

```diff
@@ -3,47 +3,196 @@ id: realms
+[Banker](./standard-library/banker.md) module
+- They can be part of a transaction call stack, as a caller or a callee
+- They can be with or without code - EOAs, or smart contracts
 
-A realm refers to a specific instance of a smart contract that can be written
-in [Gno](./gno-language.md). The potentials of realms are endless - you can create virtually any
-application in your mind with built-in composability,
-transparency, and censorship resistance. Here are some ideas of what you can build with realms:
+Realms are represented by a `Realm` type in Gno:
+```go
+type Realm struct {
+    addr    Address // Gno address in the bech32 format
+    pkgPath string  // realm's path on-chain
+}
+```
+The full Realm API can be found under the 
+[reference section](../reference/standard-library/std/realm.md).
+
+## Smart Contract Realms
+Often simply called `realms`, Gno smart contracts contain Gno code and exist
+on-chain at a specific package path. A package path is the defining identifier
+of a realm, while its address is derived from it.
+
+As opposed to [packages](./packages.md), realms are stateful, meaning they keep
+their state between transactions calls. In practice, global variables in the
+realm code are automatically persisted after a transaction has been executed,
+resulting in the fact that developers do not need to bother with the intricacies 
+of state management and persistence, like they do with other languages.
+
+### On-chain paths
+Since Gno.land is built for full transparency and auditability, all on-chain Gno
+code is open-sourced. You can view realm code by simply going to its path in
+your web browser. For example, to take a look at the `gno.land/r/demo/users` realm,
+used for user registration, by visiting
+[`gno.land/r/demo/users`](https://gno.land/r/demo/users/users.gno).
+
+:::info
+Depending on the network, the realm domain might change. Currently,
+the `gno.land/` domain (and all of its subdomains, such as `r/`) is pointing to
+the [Portal Loop](./portal-loop.md) testnet endpoint, which is subject
+to change. To view realms on the `test3` network (depr.), prepend `test3` to
+the domain: [`test3.gno.land/r/demo/users`](https://test3.gno.land/r/demo/users).
+:::
+
+[//]: # (Learn more about package paths & allowed namespaces [here].)
+
+To learn how to write a realm, see [How to write a simple Gno Smart Contract](../how-to-guides/simple-contract.md).
+
+## Externally Owned Accounts (EOAs)
+EOAs, or simply `user realms`, are Gno addresses generated from a BIP39 mnemonic
+phrase in a key management application, such as
+[`gnokey`](../gno-tooling/cli/gnokey.md), and [Adena](https://adena.app).
+
+Currently, EOAs are the only realms that can initiate a transaction. They can do
+this by calling any of the possible messages in Gno.land, such as 
+[Call](../gno-tooling/cli/gnokey.md#call),
+[AddPackage](../gno-tooling/cli/gnokey.md#addpkg),
+[Send](../gno-tooling/cli/gnokey.md#send), or Run.
```

Comment:
Again, see my previous comment about EOAs

Reply:
Please reference [this comment](https://github.com/gnolang/gno/pull/1469#discussion_r1552021046).